manifests/03-clusterrole.yaml

```diff
@@ -316,6 +316,36 @@ roleRef:
   kind: Role
   name: insights-operator
 subjects:
-- kind: ServiceAccount
-  name: operator
+  - kind: ServiceAccount
+    name: operator
+    namespace: openshift-insights
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: openshift-insights-secrets-full-access
   namespace: openshift-insights
+rules:
+  - apiGroups:
+      - ''
+    resources:
+      - secrets
+    verbs:
+      - create
+      - get
+      - watch
+      - list
+      - delete
```

Comment:
So this was wrong in our previous version? I see we had:
```
rules:
- apiGroups:
  - ""
  resources:
  - secrets
  verbs:
  - get
  - list
  - create
  - update
  - delete
```

Reply:
Hm, the previous one also works when manually applied. I gotta research more how manifests are applied :D

Reply:
It's weird...your update works locally fine for me (after applying the manifest). Isn't it all about the kubeConfig vs protoKubeConfig? I would like to fully understand TBH, because there is basically the same problem (creating/updating a secret - just in different namespace) in https://github.com/openshift/insights-operator/pull/375 and from my local testing, it seems it doesn't require any additional permissions.......

Reply:
> Isn't it all about the kubeConfig vs protoKubeConfig

Hm, not sure

Reply:
It's not...I just verified.